.github/workflows/semantic-pr.yaml

```diff
@@ -0,0 +1,24 @@
+name: "Semantic Pull Request"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+permissions:
+  # write permission is required for the wip feature to update PR status to pending when [WIP] is in the title
+  pull-requests: write
+
+jobs:
+  main:
+    name: Validate a PR title is following the conventional commits
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@0723387faaf9b38adef4775cd42cfd5155ed6017 # v5.5.3, to avoid Unpinned tag for 3rd party Action in workflow from CodeQL
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
```

Comment:
I'm not convinced that we need this feature

Reply:
yes. checked already the draft, github native, feature is used -- didn't notice that